packages/snaps-jest/src/helpers.ts

```diff
@@ -222,6 +223,30 @@ export async function installSnap<
     });
   };
 
+  const onSignature = async (request: unknown): Promise<SnapResponse> => {
```

Comment:
`onTransaction` and `onCronjob` are defined separately because they are used twice (once for the deprecated function name). Everything else can just be inlined in the object below.